pennylane/qnn/cost.py

```diff
@@ -0,0 +1,17 @@
+import pennylane as qml
+
+
+class MSECost:
+    r"""Allows users to combine an ansatz circuit with some target observables and then
+    derive a cost function from their expectation values."""
+
+    def __init__(
+            self, ansatz, observables, device, interface='autograd', diff_method='best', **kwargs
+    ):
+        # The default value for the 'measure' parameter is 'expval', but here it is made explicit for clarity
```

Comment:
It might be better to not specify `measure='expval'` explicitly and just use the default, as in this way users can specify the `measure` type as a keyword argument when instantiating `MSECost`.